src/components/Symptoms/Symptoms.js

```diff
@@ -0,0 +1,28 @@
+import Fieldset from "../Fieldset/Fieldset";
+
+function Symptoms({ handleChange, points }) {
+  const inputsInfo = [
+    ["erythema", "Rumień"],
+    ["edema", "Obrzęk"],
+    ["oozing-crusting", "Wysięki i strupki"],
+    ["lichenification", "Zgrubienie skóry"],
+    ["scratch-marks", "Przeczosy (uszkodzenia od drapania)"],
+    ["dryness", "Suchość (na obszarach niezajętych zmianami zapalnymi"],
+  ];
+
+  return (
+    <>
+      <section>
+        <h2>Część B: Intensywność objawów</h2>
+        <p>Oceń intensywność poszczególnych objawów</p>
+        <Fieldset
```

Comment:
To bym inaczej nazwała, bo to jest tak na prawdę formularz z symptomamy:
```suggestion
        <SymptomsForm
```

Reply:
done